spec/unit/mixin/powershell_out_spec.rb

```diff
@@ -44,6 +44,17 @@
       ).and_return(ret)
       expect(object.powershell_out("Get-Process", timeout: 600)).to eql(ret)
     end
+
+    context "when double quote is passed in the powershell command" do
+      it "passes if double quote is appended with single escape" do
+        result = object.powershell_out("Write-Verbose \"Some String\" -Verbose")
+        expect(result.stderr).to be == ""
```

Comment:
Not a blocker, but you might as well test stdout to make sure the command emits the expected outpit to be 100% sure the command ran as expected.